spec/features/tslr_claim_spec.rb

```diff
@@ -159,4 +159,116 @@
     visit claim_path("qts-year")
     expect(page).to have_current_path(root_path)
   end
+
+  context "editing a response" do
+    let(:claim_school) { create(:school, :tslr_eligible, name: "Claim School") }
+    let(:current_school) { create(:school, :tslr_eligible, name: "Current School") }
+
+    let(:claim) do
+      claim = create(:tslr_claim,
+        :eligible_and_submittable,
+        complete: true,
+        employment_status: :different_school,
+        claim_school: claim_school)
+      claim.current_school = current_school
```

Comment:
Could this be simplified by including it in the `create`?

Reply:
This gave me all sorts of bother when I tried to do it in the create - for some reason the `current_school` wasn't taking. I couldn't figure out why, so decided to do it this way.

Reply:
I see. That will be because of `before_save :update_current_school, if: :employment_status_changed?` in the model. Maybe that should guard against this situation, though it's possible that it will never come up in real situations.

Reply:
Yeah, I don't think this will ever come up in real life. I can use `create` and skip callbacks/validations, but this will result in more code/hacks, so I think this is fine for now. I'll add a comment to point out why I'm doing it the way I'm doing for others